magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/api/ApiContract.kt

```diff
@@ -0,0 +1,11 @@
+package com.wealthfront.magellan.sample.advanced.api
+
+import com.wealthfront.magellan.sample.advanced.cerealcollection.CerealDetails
+import io.reactivex.Observable
+import retrofit2.http.GET
+
+interface ApiContract {
+
+  @GET("v1/cereals")
+  fun getCollection(): Observable<List<CerealDetails>>
```

Comment:
Could we make this a `suspend fun`? Coroutines is what's built into the library, and kind of what we recommend